README.md

```diff
@@ -16,6 +16,36 @@ For details about configuration and deployment of RBD and CephFS CSI plugins, se
 
 For example usage of RBD and CephFS CSI plugins, see examples in `examples/`.
 
+## Support Matrix
```

Comment:
Suggesting for an alternate method to present the data. I had some further edits [here](https://hackmd.io/CLLxJR16SGupJouLDaG9jA?view) but that is outside the scope of this PR, but pointing to it nevertheless for appropriate consideration here.
```suggestion
## Support Matrix

### CSI features and availability status

#### RBD Plugin

| Features | (since) CSI driver Version |
|---|---|
| Dynamically provision, de-provision RWO/RWX file mode volumes | v0.3.0 |
| Dynamically provision, de-provision RWO/RWX block mode volumes | v1.0.0 |
| Attaching volumes to workloads | v0.3.0 |
| Creating and deleting snapshot | v1.0.0 |
| Provision volumes from snapshot | v1.0.0 |
| Provision volumes from another Volume | unsupported |
| Resize volumes | unsupported |

#### CephFS Plugin

| Features | (since) CSI driver Version |
|---|---|
| Dynamically provision, de-provision RWO/RWX file mode volumes  | v0.3.0 |
| Dynamically provision, de-provision RWO/RWX block mode volumes  | unsupported |
| Attaching volumes to workloads | v0.3.0 |
| Creating and deleting snapshot | unsupported |
| Provision volumes from snapshot | unsupported |
| Provision volumes from another Volume | unsupported |
| Resize volumes | unsupported |
```

Reply:
I think the initial version was closer to this, except for the kubernetes version column and some feature names. Have done some changes accordingly.